### ISSUE TITLE
Installation problem : permission denied

### ISSUE THREAD

**Issue by cloud9350** [meefik/linuxdeploy#357]
Status
```
[00:48:59] Updating configuration file ... done
[00:48:59] >>> status
[00:48:59] Linux Deploy: 1.5.5-184
[00:48:59] BusyBox: v1.24.1-meefik
[00:48:59] Device: LG-F260S
[00:48:59] Android: 4.4.4
[00:48:59] Architecture: armv7l
[00:48:59] Kernel: 3.4.103-perf
[00:48:59] Memory: 346/1772 MB
[00:48:59] Swap: 0/0 MB
[00:48:59] SELinux: yes
[00:48:59] Loop devices: yes
[00:48:59] Support binfmt_misc: no
[00:48:59] Supported FS: ext2 ext3 ext4 f2fs fuseblk texfat vfat 
[00:48:59] Mounted system: unknown
[00:48:59] Running services: 
[00:48:59] * SSH: no
[00:48:59] * GUI: no
[00:48:59] Mounted parts on Linux: 
[00:48:59] * /
[00:48:59] Available mount points: 
[00:48:59] * /system: 1.5/2.0 GB (ext4)
[00:48:59] * /cache: 0.8/0.8 GB (ext4)
[00:49:00] * /data: 1.4/3.8 GB (ext4)
[00:49:00] * /persist: 0.0/0.0 GB (ext4)
[00:49:00] * /firmware: 0.0/0.1 GB (vfat)
[00:49:00] * /sns: 0.0/0.0 GB (ext4)
[00:49:00] * /ve: 0.0/0.0 GB (ext4)
[00:49:00] * /persist-lg: 0.0/0.0 GB (ext4)
[00:49:00] * /mpt: 0.0/0.0 GB (ext4)
[00:49:00] * /mnt/shell/emulated: 1.4/3.8 GB (fuse)
[00:49:00] * /mnt/shell/emulated/0: 1.4/3.8 GB (fuse)
[00:49:00] * /data/sdext2: 4.8/5.9 GB (ext4)
[00:49:00] * /storage/sdcard1: 0.0/0.0 GB (fuse)
[00:49:00] * /data/media/0/Android/data/com.cloudmosa.puffinFree: 4.8/5.9 GB (ext4)
[00:49:00] Available partitions: 
[00:49:01] * /dev/block/mmcblk1p1: 1558 MB (HPFS/NTFS)
[00:49:01] * /dev/block/mmcblk1p2: 6404 MB (Linux)
[00:49:01] <<< status
```

Installation Log (Enable debug mode + trace mode)
```
[23:28:17] Updating configuration file ... done
[23:28:17] + PATH=/data/data/ru.meefik.linuxdeploy/files/bin:/data/data/ru.meefik.busybox/files/bin:/sbin:/vendor/bin:/system/sbin:/system/bin:/system/xbin
[23:28:17] + printf >>> install
[23:28:17] >>> install
[23:28:17] + linuxdeploy -t install
[23:28:17] + install_container
[23:28:17] + prepare_container
[23:28:17] + container_mounted
[23:28:17] + is_mounted /data/local/linux
[23:28:17] + local mount_point=/data/local/linux
[23:28:17] + [ -n /data/local/linux ]
[23:28:17] + grep -q  /data/local/linux  /proc/mounts
[23:28:17] Checking installation path ... done
[23:28:17] Mounting partitions: 
[23:28:17] / ... + 
[23:28:17] + return 1
[23:28:17] + msg -n Checking installation path ... 
[23:28:17] + echo -n Checking installation path ... 
[23:28:17] + [ -e /data/sdext2/ubuntu -a ! -d /data/sdext2/ubuntu ]
[23:28:17] + msg done
[23:28:17] + echo done
[23:28:17] + [ directory = file ]
[23:28:17] + [ directory = file -o directory = partition ]
[23:28:17] + [ directory = ram ]
[23:28:17] + return 0
[23:28:17] + [ 0 -ne 0 ]
[23:28:17] + mount_container root binfmt_misc
[23:28:17] + [ 2 -eq 0 ]
[23:28:17] + msg Mounting partitions: 
[23:28:17] + echo Mounting partitions: 
[23:28:17] + mount_part root
[23:28:17] + msg -n / ... 
[23:28:17] + echo -n / ... 
[23:28:17] + is_mounted /data/local/linux
[23:28:17] + local mount_point=/data/local/linux
[23:28:17] + [ -n /data/local/linux ]
[23:28:17] + grep -q  /data/local/linux  /proc/mounts
[23:28:17] + 
[23:28:17] + return 1
[23:28:17] + [ -d /data/local/linux ]
[23:28:17] + [ -d /data/sdext2/ubuntu ]
[23:28:17] + local mnt_opts=bind
[23:28:17] + mount -o bind /data/sdext2/ubuntu /data/local/linux
[23:28:17] + [ 0 -eq 0 ]
[23:28:17] + msg done
[23:28:17] + echo done
[23:28:17] + return 0
[23:28:17] + [ 0 -ne 0 ]
[23:28:17] + mount_part binfmt_misc
[23:28:17] + multiarch_support
[23:28:17] + local binfmt_dir=
[23:28:17] + [ -d /proc/sys/fs/binfmt_misc ]
[23:28:17] + grep -c ^binfmt_misc /proc/modules
[23:28:17] + [ -e /proc/modules -a 0 -ne 0 ]
[23:28:17] + [ -n  ]
[23:28:17] + return 1
[23:28:17] + local binfmt_dir=
[23:28:17] + [ -n  ]
[23:28:17] + return 0
[23:28:17] + [ 0 -ne 0 ]
[23:28:17] + return 0
[23:28:17] + [ 0 -ne 0 ]
[23:28:17] + msg Installing Debian-based distribution: 
[23:28:17] + echo Installing Debian-based distribution: 
[23:28:17] + local basic_packages=locales,sudo,man-db
[23:28:17] done
[23:28:17] Installing Debian-based distribution: 
[23:28:17] + set -e
[23:28:17] + DEBOOTSTRAP_DIR=/data/data/ru.meefik.linuxdeploy/files/share/debootstrap
[23:28:17] + . /data/data/ru.meefik.linuxdeploy/files/share/debootstrap/debootstrap --no-check-gpg --arch armhf --foreign --extractor=ar --include=locales,sudo,man-db trusty /data/local/linux http://ports.ubuntu.com/
[23:28:17] + VERSION=1.0.48
[23:28:17] + unset TMP TEMP TMPDIR
[23:28:17] + export PATH
[23:28:17] + [ -z /data/data/ru.meefik.linuxdeploy/files/share/debootstrap ]
[23:28:17] + DEVICES_TARGZ=/data/data/ru.meefik.linuxdeploy/files/share/debootstrap/devices.tgz
[23:28:17] + . /data/data/ru.meefik.linuxdeploy/files/share/debootstrap/functions
[23:28:17] + PROGRESS_NOW=0
[23:28:17] + PROGRESS_END=0
[23:28:17] + PROGRESS_NEXT=
[23:28:17] + PROGRESS_WHAT=
[23:28:17] + FINDDEBS_NEEDS_INDICES=false
[23:28:17] + SUPPORTED_VARIANTS=-
[23:28:17] + EXTRACTORS_SUPPORTED=dpkg-deb ar
[23:28:17] + UMOUNT_DIRS=
[23:28:17] + in_path perl
[23:28:17] + local OLD_IFS= 	
[23:28:17] 
[23:28:17] + IFS=:
[23:28:17] + [ -x /data/data/ru.meefik.linuxdeploy/files/bin/perl ]
[23:28:17] + [ -x /usr/local/sbin/perl ]
[23:28:17] + [ -x /usr/local/bin/perl ]
[23:28:17] + [ -x /usr/sbin/perl ]
[23:28:17] + [ -x /usr/bin/perl ]
[23:28:17] + [ -x /sbin/perl ]
[23:28:17] + [ -x /bin/perl ]
[23:28:17] + [ -x /data/data/ru.meefik.busybox/files/bin/perl ]
[23:28:17] + [ -x /data/data/ru.meefik.linuxdeploy/files/bin/perl ]
[23:28:17] + [ -x /data/data/ru.meefik.busybox/files/bin/perl ]
[23:28:17] + [ -x /sbin/perl ]
[23:28:17] + [ -x /vendor/bin/perl ]
[23:28:17] + [ -x /system/sbin/perl ]
[23:28:17] + [ -x /system/bin/perl ]
[23:28:17] + [ -x /system/xbin/perl ]
[23:28:17] + IFS= 	
[23:28:17] 
[23:28:17] + return 1
[23:28:17] + [ -e /usr/lib/debootstrap/pkgdetails ]
[23:28:17] + [ -e /data/data/ru.meefik.linuxdeploy/files/share/debootstrap/pkgdetails ]
[23:28:17] + PKGDETAILS=/data/data/ru.meefik.linuxdeploy/files/share/debootstrap/pkgdetails
[23:28:17] + N_EXIT_THINGS=0
[23:28:17] + trap exit_function 0
[23:28:17] + trap exit 129 1
[23:28:17] + trap error 130 INTERRUPTED "Interrupt caught ... exiting" 2
[23:28:17] + trap exit 131 3
[23:28:17] + trap exit 143 15
[23:28:17] + exec
[23:28:17] + LANG=C
[23:28:17] + USE_COMPONENTS=main
[23:28:17] + KEYRING=
[23:28:17] + DISABLE_KEYRING=
[23:28:17] + VARIANT=
[23:28:17] + ARCH=
[23:28:17] + HOST_ARCH=
[23:28:17] + HOST_OS=
[23:28:17] + KEEP_DEBOOTSTRAP_DIR=
[23:28:17] + USE_DEBIANINSTALLER_INTERACTION=
[23:28:17] + SECOND_STAGE_ONLY=
[23:28:17] + PRINT_DEBS=
[23:28:17] + CHROOTDIR=
[23:28:17] + MAKE_TARBALL=
[23:28:17] + KEEP_DEBOOTSTRAP_DIR=
[23:28:17] + EXTRACTOR_OVERRIDE=
[23:28:17] + UNPACK_TARBALL=
[23:28:17] + ADDITIONAL=
[23:28:17] + EXCLUDE=
[23:28:17] + VERBOSE=
[23:28:17] + CERTIFICATE=
[23:28:17] + CHECKCE+ [ 1 -ne 0 ]
[23:28:17] + return 1
[23:28:17] + printf <<< install
[23:28:17] 
[23:28:17] + exit 0
[23:28:17] RTIF=
[23:28:17] + PRIVATEKEY=
[23:28:17] + DEF_MIRROR=http://ftp.us.debian.org/debian
[23:28:17] + export LANG USE_COMPONENTS
[23:28:17] + umask 022
[23:28:17] + RESOLVE_DEPS=true
[23:28:17] + WHAT_TO_DO=finddebs dldebs first_stage second_stage
[23:28:17] + [ -z /data/data/ru.meefik.linuxdeploy/files/share/debootstrap/pkgdetails ]
[23:28:17] + [ 9 != 0 ]
[23:28:17] + true
[23:28:17] + shift 1
[23:28:17] + DISABLE_KEYRING=1
[23:28:17] + true
[23:28:17] + [ --arch = --arch -a -n armhf ]
[23:28:17] + ARCH=armhf
[23:28:17] + shift 2
[23:28:17] + true
[23:28:17] + [  != true ]
[23:28:17] + WHAT_TO_DO=finddebs dldebs first_stage
[23:28:17] + shift
[23:28:17] + true
[23:28:17] + [ --extractor=ar = --extractor -a -n --include=locales,sudo,man-db ]
[23:28:17] + [ --extractor=ar != ar ]
[23:28:17] + EXTRACTOR_OVERRIDE=ar
[23:28:17] + shift
[23:28:17] + valid_extractor ar
[23:28:17] + local extractor=ar
[23:28:17] + [ ar = dpkg-deb ]
[23:28:17] + [ ar = ar ]
[23:28:17] + return 0
[23:28:17] + type ar
[23:28:17] + true
[23:28:17] + [ --include=locales,sudo,man-db = --include -a -n trusty ]
[23:28:17] + [ --include=locales,sudo,man-db != locales,sudo,man-db ]
[23:28:17] + ADDITIONAL=locales,sudo,man-db
[23:28:17] + shift 1
[23:28:17] + echo locales,sudo,man-db
[23:28:17] + tr ,  
[23:28:17] + ADDITIONAL=locales sudo man-db
[23:28:17] + true
[23:28:17] + break
[23:28:17] + [  = true ]
[23:28:17] + [ -z trusty ]
[23:28:17] + [ -z /data/local/linux ]
[23:28:17] + SUITE=trusty
[23:28:17] + TARGET=/data/local/linux
[23:28:17] + TARGET=/data/local/linux
[23:28:17] + [ data/local/linux = /data/local/linux ]
[23:28:17] + SCRIPT=/data/data/ru.meefik.linuxdeploy/files/share/debootstrap/scripts/trusty
[23:28:17] + [ -n  ]
[23:28:17] + [  !=  ]
[23:28:17] + in_path dpkg
[23:28:17] + local OLD_IFS= 	
[23:28:17] 
[23:28:17] + IFS=:
[23:28:17] + [ -x /data/data/ru.meefik.linuxdeploy/files/bin/dpkg ]
[23:28:17] + [ -x /usr/local/sbin/dpkg ]
[23:28:17] + [ -x /usr/local/bin/dpkg ]
[23:28:17] + [ -x /usr/sbin/dpkg ]
[23:28:17] + [ -x /usr/bin/dpkg ]
[23:28:17] + [ -x /sbin/dpkg ]
[23:28:17] + [ -x /bin/dpkg ]
[23:28:17] + [ -x /data/data/ru.meefik.busybox/files/bin/dpkg ]
[23:28:17] + IFS= 	
[23:28:17] 
[23:28:17] + return 0
[23:28:17] + dpkg --print-architecture
[23:28:17] + in_path udpkg
[23:28:17] + local OLD_IFS= 	
[23:28:17] 
[23:28:17] + IFS=:
[23:28:17] + [ -x /data/data/ru.meefik.linuxdeploy/files/bin/udpkg ]
[23:28:17] + [ -x /usr/local/sbin/udpkg ]
[23:28:17] + [ -x /usr/local/bin/udpkg ]
[23:28:17] + [ -x /usr/sbin/udpkg ]
[23:28:17] + [ -x /usr/bin/udpkg ]
[23:28:17] + [ -x /sbin/udpkg ]
[23:28:17] + [ -x /bin/udpkg ]
[23:28:17] + [ -x /data/data/ru.meefik.busybox/files/bin/udpkg ]
[23:28:17] + [ -x /data/data/ru.meefik.linuxdeploy/files/bin/udpkg ]
[23:28:17] + [ -x /data/data/ru.meefik.busybox/files/bin/udpkg ]
[23:28:17] + [ -x /sbin/udpkg ]
[23:28:17] + [ -x /vendor/bin/udpkg ]
[23:28:17] + [ -x /system/sbin/udpkg ]
[23:28:17] + [ -x /system/bin/udpkg ]
[23:28:17] + [ -x /system/xbin/udpkg ]
[23:28:17] + IFS= 	
[23:28:17] 
[23:28:17] + return 1
[23:28:17] + [ -e /data/data/ru.meefik.linuxdeploy/files/share/debootstrap/arch ]
[23:28:17] + HOST_OS=
[23:28:17] + [ -z  ]
[23:28:17] + uname
[23:28:17] + HOST_OS=linux
[23:28:17] + [ -z armhf ]
[23:28:17] + [ -z armhf ]
[23:28:17] + [ -z linux ]
[23:28:17] + [ linux = kfreebsd ]
[23:28:17] + [ linux = freebsd ]
[23:28:17] + [ /data/local/linux = / ]
[23:28:17] + doing_variant scratchbox
[23:28:17] + [ scratchbox =  ]
[23:28:17] + [ scratchbox = - ]
[23:28:17] + return 1
[23:28:17] + CHROOT_CMD=chroot /data/local/linux
[23:28:17] + [ -z  ]
[23:28:17] + SHA_SIZE=256
[23:28:17] + in_path sha256sum
[23:28:17] + local OLD_IFS= 	
[23:28:17] 
[23:28:17] + IFS=:
[23:28:17] + [ -x /data/data/ru.meefik.linuxdeploy/files/bin/sha256sum ]
[23:28:17] + [ -x /usr/local/sbin/sha256sum ]
[23:28:17] + [ -x /usr/local/bin/sha256sum ]
[23:28:17] + [ -x /usr/sbin/sha256sum ]
[23:28:17] + [ -x /usr/bin/sha256sum ]
[23:28:17] + [ -x /sbin/sha256sum ]
[23:28:17] + [ -x /bin/sha256sum ]
[23:28:17] + [ -x /data/data/ru.meefik.busybox/files/bin/sha256sum ]
[23:28:17] + IFS= 	
[23:28:17] 
[23:28:17] + return 0
[23:28:17] + DEBOOTSTRAP_CHECKSUM_FIELD=SHA256
[23:28:17] + export ARCH SUITE TARGET CHROOT_CMD SHA_SIZE DEBOOTSTRAP_CHECKSUM_FIELD
[23:28:17] + am_doing_phase first_stage second_stage
[23:28:17] + local x
[23:28:17] + echo  finddebs dldebs first_stage 
[23:28:17] + grep -q  first_stage 
[23:28:17] + return 0
[23:28:17] + in_path id
[23:28:17] + local OLD_IFS= 	
[23:28:17] 
[23:28:17] + IFS=:
[23:28:17] + [ -x /data/data/ru.meefik.linuxdeploy/files/bin/id ]
[23:28:17] + [ -x /usr/local/sbin/id ]
[23:28:17] + [ -x /usr/local/bin/id ]
[23:28:17] + [ -x /usr/sbin/id ]
[23:28:17] + [ -x /usr/bin/id ]
[23:28:17] + [ -x /sbin/id ]
[23:28:17] + [ -x /bin/id ]
[23:28:17] + IFS= 	
[23:28:17] 
[23:28:17] + return 0
[23:28:17] + id -u
[23:28:17] + [ uid=0(root) gid=0(root) context=u:r:init_shell:s0 -ne 0 ]
[23:28:17] ash: gid=0(root): unknown operand
[23:28:17] + check_sane_mount /data/local/linux
[23:28:17] + mkdir -p /data/local/linux
[23:28:17] + mknod /data/local/linux/test-dev-null c 1 3
[23:28:17] + echo test
[23:28:17] + rm -f /data/local/linux/test-dev-null
[23:28:17] + SH=/bin/sh
[23:28:17] + [ -x /bin/sh ]
[23:28:17] + cat
[23:28:17] + chmod +x /data/local/linux/test-exec
[23:28:17] Bad mode
[23:28:17] + /data/local/linux/test-exec
[23:28:17] /data/data/ru.meefik.linuxdeploy/files/bin/linuxdeploy: /data/data/ru.meefik.linuxdeploy/files/share/debootstrap/debootstrap: line 1: /data/local/linux/test-exec: Permission denied
[23:28:17] + rm -f /data/local/linux/test-exec
[23:28:17] + return 1
[23:28:17] + error 1 NOEXEC Cannot install into target '/data/local/linux' mounted with noexec or nodev
[23:28:17] + local err=1
[23:28:17] + local name=NOEXEC
[23:28:17] + local fmt=Cannot install into target '/data/local/linux' mounted with noexec or nodev
[23:28:17] + shift
[23:28:17] + shift
[23:28:17] + shift
[23:28:17] + [  ]
[23:28:17] + printf E: Cannot install into target '/data/local/linux' mounted with noexec or nodev\n
[23:28:17] E: Cannot install into target '/data/local/linux' mounted with noexec or nodev
[23:28:17] + exit 1
[23:28:17] + exit_function
[23:28:17] + local n=0
[23:28:17] + [ 0 -lt 0 ]
[23:28:17] + N_EXIT_THINGS=0
[23:28:17] <<< install
```

Settings (For the convenience of developers changing the language)
![Imgur](http://i.imgur.com/8HqKrBL.jpg)
![Imgur](http://i.imgur.com/GTepOTl.jpg)
![Imgur](http://i.imgur.com/leXG8mA.jpg)

I delete chroot folder, and data folder, reinstall application & update env, chmod 777 folders, rebooting, "SMARTPHONE FOTMAT + REINSTALL SYSTEM"
But can't solve issue.

How to fix this problem?

**Comment by meefik** [comment]
This is a full log?

**Comment by cloud9350** [comment]
@meefik This turn off debug mode log
```
[22:16:15] Updating configuration file ... done
[22:16:15] >>> install
[22:16:15] Checking installation path ... done
[22:16:15] Mounting partitions: 
[22:16:15] / ... done
[22:16:15] Installing Debian-based distribution: 
[22:16:15] ash: gid=0(root): unknown operand
[22:16:15] Bad mode
[22:16:15] /data/data/ru.meefik.linuxdeploy/files/bin/linuxdeploy: /data/data/ru.meefik.linuxdeploy/files/share/debootstrap/debootstrap: line 475: /data/local/linux/test-exec: Permission denied
[22:16:15] E: Cannot install into target '/data/local/linux' mounted with noexec or nodev
```

**Comment by meefik** [comment]
Try remove busybox and install again from BusyBox app interface.

**Comment by cloud9350** [comment]
@meefik I reinstall busybox and start linux deploy, But no solve issue.

```
[23:23:02] Updating configuration file ... done
[23:23:02] >>> install
[23:23:02] The container is already mounted.
[23:23:02] <<< install
[23:23:04] Updating configuration file ... done
[23:23:04] >>> stop
[23:23:04] Stopping services: 
[23:23:04] SSH ... done
[23:23:04] VNC ... fail
[23:23:04] Release resources ... done
[23:23:05] Unmounting partitions: 
[23:23:05] / ... done
[23:23:05] Disassociating loop device ... done
[23:23:05] <<< stop
[23:23:06] Updating configuration file ... done
[23:23:06] >>> install
[23:23:06] Checking installation path ... done
[23:23:06] Mounting partitions: 
[23:23:06] / ... done
[23:23:06] Installing Debian-based distribution: 
[23:23:06] ash: gid=0(root): unknown operand
[23:23:07] Bad mode
[23:23:07] /data/data/ru.meefik.linuxdeploy/files/bin/linuxdeploy: /data/data/ru.meefik.linuxdeploy/files/share/debootstrap/debootstrap: line 475: /data/local/linux/test-exec: Permission denied
[23:23:07] E: Cannot install into target '/data/local/linux' mounted with noexec or nodev
[23:23:07] <<< install
```

**Comment by meefik** [comment]
If system BusyBox not working:
1) Install compatible BusyBox (https://github.com/meefik/busybox/releases)
2) Run BusyBox app.
3) Run Linux Deploy app and change "Settings -> BusyBox directory" to /data/data/ru.meefik.busybox/files/bin

**Comment by meefik** [comment]
Also see #358

**Comment by cloud9350** [comment]
@meefik Not working
![Imgur](http://i.imgur.com/lh8rzCm.jpg)
![Imgur](http://i.imgur.com/r5PugRD.jpg)

**Comment by meefik** [comment]
Run "Settings -> Update ENV".

**Comment by cloud9350** [comment]
@meefik Already. Not working.

P.S - Working it Linux Deploy 1.5.3 version...

**Comment by cloud9350** [comment]
@meefik I update Installation Log.
Please read this. (First comment)

**Comment by cloud9350** [comment]
@meefik
I delete chroot, data folder, and reinstall Linux deploy, and remove env, update env, install.

Output
```
[23:41:42] Updating environment ... done
[23:41:49] Updating configuration file ... done
[23:41:49] + PATH=/data/data/ru.meefik.linuxdeploy/files/bin:/system/xbin:/sbin:/vendor/bin:/system/sbin:/system/bin:/system/xbin
[23:41:49] + printf >>> install
[23:41:49] >>> install
[23:41:49] + linuxdeploy -t install
[23:41:49] + container_install
[23:41:49] + container_prepare
[23:41:49] + container_mounted
[23:41:49] + is_mounted /data/local/linux
[23:41:49] + local mount_point=/data/local/linux
[23:41:49] + [ -n /data/local/linux ]
[23:41:49] + grep -q  /data/local/linux  /proc/mounts
[23:41:49] + 
[23:41:49] + return 1
[23:41:49] + msg -n Checking installation path ... 
[23:41:49] + echo -n Checking installation path ... 
[23:41:49] + [ -e /data/sdext2/ubuntu -a ! -d /data/sdext2/ubuntu ]
[23:41:49] + msg done
[23:41:49] + echo done
[23:41:49] + [ directory = file ]
[23:41:49] + [ directory = file -o directory = partition ]
[23:41:49] + [ directory = ram ]
[23:41:49] + return 0
[23:41:49] + [ 0 -ne 0 ]
[23:41:49] + container_mount root binfmt_misc
[23:41:49] + [ 2 -eq 0 ]
[23:41:49] + msg Mounting partitions: 
[23:41:49] + echo Mounting partitions: 
[23:41:49] + mount_part root
[23:41:49] + msg -n / ... 
[23:41:49] + echo -n / ... 
[23:41:49] + is_mounted /data/local/linux
[23:41:49] + local mount_point=/data/local/linux
[23:41:49] + [ -n /data/local/linux ]
[23:41:49] + grep -q  /data/local/linux  /proc/mounts
[23:41:49] + 
[23:41:49] + return 1
[23:41:49] + [ -d /data/local/linux ]
[23:41:49] + [ -d /data/sdext2/ubuntu ]
[23:41:49] + local mnt_opts=rw,relatime
[23:41:49] + mount -o rw,relatime /data/sdext2/ubuntu /data/local/linux
[23:41:49] Checking installation path ... done
[23:41:49] Mounting partitions: 
[23:41:49] / ... fail
[23:41:49] Usage: mount [-r] [-w] [-o options] [-t type] device directory
[23:41:49] + [ 1 -eq 0 ]
[23:41:49] + msg fail
[23:41:49] + echo fail
[23:41:49] + return 1
[23:41:49] + [ 1 -ne 0 ]
[23:41:49] + return 1
[23:41:49] + [ 1 -ne 0 ]
[23:41:49] + return 1
[23:41:49] + printf <<< install
[23:41:49] <<< install
```

**Comment by meefik** [comment]
Now I will prepare a new version of LD, it will be necessary to check.

**Comment by cloud9350** [comment]
@meefik I found fix
Please check pull request

**Comment by cloud9350** [comment]
This issue is fixed (version 1.5.5)
